aws-alexa/src/test/groovy/io/micronaut/aws/alexa/locale/HandlerInputLocaleResolverSpec.groovy

```diff
@@ -0,0 +1,52 @@
+package io.micronaut.aws.alexa.locale
+
+import com.amazon.ask.dispatcher.request.handler.HandlerInput
+import com.amazon.ask.model.Request
+import io.micronaut.core.util.LocaleResolver
+import io.micronaut.test.extensions.spock.annotation.MicronautTest
+import jakarta.inject.Inject
+import spock.lang.Specification
+import spock.lang.Unroll
+
+@MicronautTest(startApplication = false)
+class HandlerInputLocaleResolverSpec extends Specification {
+
+    @Inject
+    LocaleResolver<HandlerInput> localeResolver
+
+    @Unroll
+    void "resolve locale from handler input"(String languageCode, String language, Locale expected) {
```

Comment:
Did you mean to use `language` in the test name?

Reply:
I have removed it https://github.com/micronaut-projects/micronaut-aws/pull/1483/commits/62deb06d76473a8a9231a78ff604d4fc30da81e3